apps/labs/posts/introducing-intern-cohort-2022.md

```diff
@@ -0,0 +1,31 @@
+---
+title: 'Introducing the 2022 Interns Cohort'
+author: noa-tamir
+published: August 7, 2022
+description: 'Quansight Labs is delighted to welcome its second cohort of 6 interns, who will work on a variety of open source projects and tasks'
+category: [Community, OSS Experience, PyData ecosystem]
+featuredImage:
+src: /posts/introducing-intern-cohort-2022/feature.svg
+alt: 'An illustration of a black hand and a white one coming towards each other to pass a business card with the logo of Quansight Labs'
+hero:
+imageSrc: /posts/introducing-intern-cohort-2022/hero_intern_cohort.svg
+imageAlt: 'Circular images of all the interns and their mentors connected with lines indicating who is working with whom. Each image is labeled with their first name.'
+---
+
+We recently had the great pleasure of welcoming our second cohort of interns at Labs! The goal of our internship program is to provide them the opportunity to connect with open source communities and leaders, and grow professionally while developing solutions for the larger scientific open source ecosystem. Without further ado, allow us to introduce this wonderful new team and their projects!
+
+[Asmi Jafar](https://www.github.com/asmijafar20), based in Aligarh, India, is a final year student of masters in Computer Applications at Aligarh Muslim University (AMU). Asmi is profoundly influenced by technology and the various ways it can be used to express art. She is interested in contributing to open source projects to gain skills, and learning new things. Asmi has previously contributed to a variety of projects in the Moja Global community while completing her Outreachy internship. In her spare time, she loves to read good books, paint, and dance. Here's a quote she chose to share from one of her favorite works, The Last Lecture, by Jeffrey Zaslow and Randy Pausch:
+| The Brick walls are there for a reason, they are not there to keep us out. The brick walls are there to give us a chance to show how badly we want something.
```

Comment:
This is not correctly formatted in the preview, not sure why!

Reply:
Good catch! Fixed.